### PR TITLE
bz2062068 updating CA bundle certificates

### PR DESCRIPTION
**Pull request by sheriff-rh**
Changes apply to 4.6+
https://bugzilla.redhat.com/show_bug.cgi?id=2062068

Build preview (VPN required):
http://file.rdu.redhat.com/antaylor/06-03-22/bz2062068/networking/enable-cluster-wide-proxy.html
http://file.rdu.redhat.com/antaylor/06-03-22/bz2062068/security/certificates/updating-ca-bundle.html#ca-bundle-understanding_updating-ca-bundle

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@zhaozhanqi  PTAL

File: modules/ca-bundle-understanding.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/updating-ca-bundle.adoc
+
+:_content-type: SNIPPET
+[id="ca-bundle-understanding_{context}"]
+= Understanding the CA Bundle certificate
+
+Proxy certificates allow users to specify one or more custom certificate authority (CA) certificates used by platform components when making egress connections.
```
Comment (opayne1) [inline modules/ca-bundle-understanding.adoc:9]: This sentence confused me a bit with certificate mentioned a few times. What do you think of removing that last use of certificates?
```suggestion
Proxy certificates allow users to specify one or more custom certificate authority (CA) used by platform components when making egress connections.
```

**Review by opayne1 — COMMENTED** [review]
Just one little nit if that doesn't change any meaning to the sentence :)

File: modules/ca-bundle-understanding.adoc (hunk 1 above)
Comment (sheriff-rh) [reply modules/ca-bundle-understanding.adoc]: Good catch! Change committed; pushing.

**Pull request merged**